src/session8/homework/ChristmasWishlist.java

```diff
@@ -0,0 +1,26 @@
+package session8.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ChristmasWishlist {
+
+    public static void main(String[] args) {
+        List<String> wishlist = new ArrayList<>();
+        wishlist.add("Lego Millennium Falcon");
+        wishlist.add("Potatoes");
+        wishlist.add("IT School Java Course");
+        wishlist.add("Rum");
+        wishlist.add("Eye patch");
+        wishlist.add("Wooden Leg");
+
+        System.out.println(wishlist);
+
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
From now on, we should try to pay more attention to how we structure our code, and if we can decouple some logic from one method, and it makes sense to do so, we should do it.
By the definition of what clean code means, we should have only one responsibility for a method/function. This will not be possible every time, but we should have this statement in mind.

In this case, we can split our for loop and string builder into another method.

This comment stands for all the scenarios where there is more than one step into a method.